packages/core/directives/map.ts

```diff
@@ -358,6 +358,11 @@ export class AgmMap implements OnChanges, OnInit, OnDestroy {
    */
   @Output() mapReady: EventEmitter<any> = new EventEmitter<any>();
 
+  /**
+   * This event is fired when the visible tiles have finished loading.
+   */
+  @Output() tilesLoaded: EventEmitter<any> = new EventEmitter<any>();
```

Comment:
please make it a `void` event, not `any`

Reply:
Good catch! Done.